kitsune/questions/jinja2/questions/includes/aaq_macros.html

```diff
@@ -95,34 +95,23 @@ <h2 class="card--title has-bottom-margin">
 
       {{ _('Ask a Question') }}
     </h2>
+    {% set link_detail = aaq_context.product_slug if aaq_context else topic %}
+    {% set has_support = aaq_context.has_ticketing_support if aaq_context else False %}
+    {% set has_forum = aaq_context.has_public_forum if aaq_context else False %}
+    
     {% if aaq_context %}
-      {% set link_detail = aaq_context.product_slug %}
       {% if request.user.is_authenticated %}
-        {% if aaq_context.has_ticketing_support %}
-          <p> {{ _('Still need help? Continue to contact our support team.') }}</p>
-        {% elif aaq_context.has_public_forum %}
-          <p>{{ _('Still need help? Continue to ask your question on our forums.') }}</p>
-        {% endif %}
+        <p>{{ _('Still need help? Continue to ' + ('contact our support team' if has_support else 'ask your question on our forums') + '.') }}</p>
```

Comment:
I think there's a valid point for the string extraction process since this is evaluated dynamically. That said, I am not sure how `format` can help in this case though. I am going to separate the strings in multiple lines.